cheroot/workers/threadpool.py

```diff
@@ -151,12 +151,20 @@ def __init__(
             server (cheroot.server.HTTPServer): web server object
                 receiving this request
             min (int): minimum number of worker threads
-            max (int): maximum number of worker threads
+            max (int): maximum number of worker threads (-1 for no max)
             accepted_queue_size (int): maximum number of active
                 requests in queue
             accepted_queue_timeout (int): timeout for putting request
                 into queue
+
+        Raises:
+            ValueError: if the min/max values are invalid
         """
+        if min < 1:
+            raise ValueError('min must be > 0')
```

Comment:
The WPS226 linting violation (https://results.pre-commit.ci/run/github/16620627/1670250782.Xd5wEFEFTPKTk0zOvjQ30Q) hinted me that it could be useful to have the broken values presented to the users:
```suggestion
            raise ValueError(f'{min=} must be > 0')
```

Reply:
This `{x=}` f-string syntax is only available in Python3.8+ and this package supports v3.6 - I've used the more verbose `x={x}`syntax instead.